projects/dsp-ui/src/lib/viewer/views/property-view/property-view.component.spec.ts

```diff
@@ -80,17 +83,36 @@ class TestDisplayValueComponent {
 
 }
 
+/**
+ * Test host component to simulate child component, here add-value.
+ */
+@Component({
+    selector: `dsp-add-value`,
+    template: ``
+  })
+  class TestAddValueComponent {
+
+    @Input() parentResource: ReadResource;
+    @Input() resourcePropertyDefinition: ResourcePropertyDefinition;
+
+  }
+
 describe('PropertyViewComponent', () => {
   let testHostComponent: TestPropertyParentComponent;
   let testHostFixture: ComponentFixture<TestPropertyParentComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
+      imports: [
+        MatIconModule,
+      ],
       declarations: [
         TestPropertyParentComponent,
         TestDisplayValueComponent,
+        TestAddValueComponent,
         PropertyViewComponent
-      ]
+      ],
+      providers: [EventBusService]
```

Comment:
Could you mock and spy on the event bus service and see if the property view component reacts correctly to a newly created value, e.g. `hideAddValueForm` is called?

Reply:
is there a need to mock the service? I think it would be enough to test is like I did in the resource-view spec file

Reply:
Yes, I think if the mock is as complex as the actual thing, there is not much benefit. Maybe a thing to be covered in https://dasch.myjetbrains.com/youtrack/issue/DSP-98?

Reply:
done in 2797c07